client/lib/discounts/active-discounts.js

```diff
@@ -54,4 +55,22 @@ export default [
 		plansPageNoticeText:
 			'Improve your SEO, branding, credibility, and even word-of-mouth marketing with a custom domain. All plan upgrades include a free domain name of your choice.',
 	},
+	{
+		name: 'august20',
+		startsAt: new Date( 2018, 7, 6, 0, 0, 0 ),
+		endsAt: new Date( 2018, 7, 11, 0, 0, 0 ),
+		nudgeText: translate( '20% Off All Plans' ),
+		ctaText: translate( 'UPGRADE' ),
+		plansPageNoticeText: translate(
+			'Enter coupon code “AUGUST20” during checkout to claim your 20% discount.'
+		),
+		targetPlans: [
+			{ type: TYPE_FREE, group: GROUP_WPCOM },
+			{ type: TYPE_PERSONAL, group: GROUP_WPCOM },
+			{ type: TYPE_PREMIUM, group: GROUP_WPCOM },
+			{ type: TYPE_FREE, group: GROUP_JETPACK },
+			{ type: TYPE_PERSONAL, group: GROUP_JETPACK },
+			{ type: TYPE_PREMIUM, group: GROUP_JETPACK },
+		],
```

Comment:
Just a note: since each item in `targetPlans` is matched against current plan, we may either specify `group` explicitly and list all plans for both WPCOM and JETPACK, or omit the `group` property and use the following, more relaxed declaration instead:
```
			{ type: TYPE_FREE },
			{ type: TYPE_PERSONAL },
			{ type: TYPE_PREMIUM },
```
This will be matched for both jetpack and wpcom